scripts/dev/e2e.py

```diff
@@ -72,6 +76,29 @@ def _prepare_testrunner_environment():
     )
 
 
+def create_kube_config():
+    """Replicates the local kubeconfig file (pointed at by KUBECONFIG),
+    as a ConfigMap."""
+    corev1 = client.CoreV1Api()
+    print("Creating kube-config ConfigMap")
+
+    svc = corev1.read_namespaced_service("kubernetes", "default")
+    kube_config = os.getenv("KUBECONFIG")
+    with open(kube_config) as fd:
+        kube_config = yaml.safe_load(fd.read())
+
+    kube_config["clusters"][0]["cluster"]["server"] = "https://" + svc.spec.cluster_ip
+    kube_config = yaml.safe_dump(kube_config)
+    data = {"kubeconfig": kube_config}
+    config_map = client.V1ConfigMap(
+        metadata=client.V1ObjectMeta(name="kube-config"), data=data
+    )
+
+    ignore_if_already_exists(
+        lambda: corev1.create_namespaced_config_map("default", config_map)
+    )
+
```

Comment:
🙏 Thanks for moving this into the python script! (can we delete it from the bash one, or is that still needed for evg?)

Reply:
Still needed for Evergreen runs

Reply:
and removed from kind setup!